### PR TITLE
Fs my weights

### PR DESCRIPTION
**Pull request by fszwedo**
My weights full component.
Component returns the div that should be placed in the overview menu. Component function takes the user object as an argument and displays the 'weights' values in a table.

### CODE REVIEW

**Comment by fszwedo** [comment]
sorry guys, I see that because I merged my branch with Robert's one to get the utils functions he added now all the files edited in his commits are marked as edited here as well - sorry for that, please just check the MyWeights files

File: Src/UIComponents/MyWeights/MyWeights.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { User } from '../../../Models/User.model';
+
+export default function myWeightsComponent(user:User):HTMLDivElement{
+    const container = createElement('div', 'my-weights-container') as HTMLDivElement;
+    const weightsTile = generateWeightTable(user.weights);
+    const header = createElement('div', 'myweights-header');
+    const headerText = document.createTextNode('My weights');
+
+    header.append(headerText);
+
```
Comment (robert-czwartosz) [inline Src/UIComponents/MyWeights/MyWeights.ts:12]: These lines could be replaced by add third argument ('My weights') to createElement.

**Review by robert-czwartosz — COMMENTED** [review]
Only one small thing is to correct.
Component looks very good.

**Review by skorpiom — APPROVED** [review]

**Comment by skorpiom** [comment]
Component looks great!

File: Src/UIComponents/MyWeights/MyWeights.ts (hunk 1 above)
Comment (fszwedo) [reply Src/UIComponents/MyWeights/MyWeights.ts]: done

**Review by robert-czwartosz — APPROVED** [review]

**Pull request merged**